desktop/e2e/tests/lib/video-recorder.js

```diff
@@ -31,6 +31,10 @@ exports.createDir = function ( dir ) {
 
 exports.isVideoEnabled = function () {
 	const video = process.env.CI;
+	// eslint-disable-next-line no-console
```

Comment:
Curious if including L34-L37 was intentional here? Intentional or was this something you used for debugging?

Reply:
Good catch!
Leftovers from when I was trying to debug video recorder, deleted.